src/radon.ts

```diff
@@ -194,6 +194,12 @@ export class AggregationTallyScript {
     )
   }
 
+  // Remove the filter from the filter's list by id
+  public deleteOperator(operatorId: number) {
```

Comment:
Why does this accept operatorId instead of operatorIndex? If the answer is "because the filter does not know its own index", then this will fail when there are two filters with the same id:

```js
filters: [
        [AggregationTallyFilter.greaterThan, 3],
        AggregationTallyFilter.mode,
        [AggregationTallyFilter.greaterThan, 3],
      ]
```

Then calling `deleteOperator(AggregationTallyFilter.greaterThan)` will always remove the first element, which may not be what we actually want.